pkg/okteto/constants/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 The Okteto Authors
```

Comment:
Why do we need this file?

Reply:
It's for documenting the package. Check out the lines after the license

It's not necessary